docs/content/tutorials/kic/ingress-walkthrough.md

```diff
@@ -9,41 +9,48 @@ docs: "DOCS-723"
 
 ## Overview
 
-Follow this tutorial to deploy the NGINX Plus Ingress Controller with NGINX Service Mesh and an example application.
+Follow this tutorial to deploy the NGINX Ingress Controller with NGINX Service Mesh and an example application.
 
 Objectives:
 
 - Deploy the NGINX Service Mesh.
-- Install NGINX Plus Ingress Controller.
+- Install NGINX Ingress Controller.
 - Deploy the example `bookinfo` app.
   - {{< fa "download" >}} {{< link "/examples/bookinfo.yaml" >}}
 - Create a Kubernetes Ingress resource for the Bookinfo application.
 
 {{< note >}}
-All communication between the NGINX Plus Ingress Controller and the Bookinfo application occurs over mTLS.
+All communication between the NGINX Ingress Controller and the Bookinfo application occurs over mTLS.
 {{< /note >}}
 
-{{< note >}} The NGINX Plus version of NGINX Plus Ingress Controller is required for this tutorial. {{< /note >}}
+{{< note >}}
+There are two versions of NGINX Ingress Controller for Kubernetes: NGINX Open Source and NGINX Plus.
```

Comment:
"NGINX Ingress Controller can be used free with NGINX Open Source and paying customers have access to NGINX Ingress Controller with NGINX Plus."

This is always difficult to explain.  And it needs to be clear it is the same product and the difference is NGINX.
I like the bullet list call out of the version below.